packages/mui-joy/src/Select/Select.tsx

```diff
@@ -125,6 +125,18 @@ const SelectRoot = styled('div', {
         '--Select-decoratorChildHeight': 'min(2.375rem, var(--Select-minHeight))',
         '--Icon-fontSize': theme.vars.fontSize.xl2,
       }),
+      ...(ownerState.variant === 'plain' && {
+        boxShadow: 'none',
+      }),
+      ...(ownerState.variant === 'outlined' && {
+        boxShadow: theme.shadow.xs,
+      }),
+      ...(ownerState.variant === 'soft' && {
+        boxShadow: theme.shadow.xs,
+      }),
+      ...(ownerState.variant === 'solid' && {
+        boxShadow: theme.shadow.xs,
+      }),
```

Comment:
```suggestion
      ...(ownerState.variant !== 'plain' && {
        boxShadow: 'none',
      }),
```
Would this be better? the downside would be if developers provide a custom variant, the shadow will be applied.

Reply:
Did you mean this?
```
      ...(ownerState.variant !== 'plain' && {
	        boxShadow: 'none',
	      }),
```
Or are you suggesting to add shadows only to plain variant?